src/Collection.php

```diff
@@ -72,16 +72,24 @@ public function endpoint_callback( \WP_REST_Request $request ) {
 	protected function loop() {
 		$data = [];
 
+		$meta = [];
+
 		$this->query = new \WP_Query( $this->args );
 		while ( $this->query->have_posts() ) {
 			$this->query->the_post();
 			$data[] = $this->format_item( $this->query->post );
+
+			if ( empty( $meta ) ) {
```

Comment:
Because we only want to fill it the first time we pass through the loop. CI is because wp-utils aren't in packagist yet, will upload now